openstack/cce/v3/nodes/testing/requests_test.go

```diff
@@ -5,18 +5,26 @@ import (
 	"net/http"
 	"testing"
 
-	fake "github.com/opentelekomcloud/gophertelekomcloud/openstack/cce/v3/common"
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/testhelper/client"
+
 	"github.com/opentelekomcloud/gophertelekomcloud/openstack/cce/v3/nodes"
 	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
 )
 
+func serviceClient() *golangsdk.ServiceClient {
+	sc := client.ServiceClient()
+	sc.ResourceBase = sc.Endpoint + "api/" + "v3/" + "projects/" + "c59fd21fd2a94963b822d8985b884673/"
+	return sc
+}
+
```

Comment:
Why have you copied `ServiceClient` to several packages?
They are literally the same function, as far as I can see.